distribution/feature-packs/server-feature-pack/src/main/resources/content/bin/migrate-standalone-ha.cli

```diff
@@ -1,3 +1,8 @@
+echo
+echo *** WARNING ***
+echo A manual migration step may be needed if start of the embed-server fails. In such case proceed to Keycloak migration guide.
+echo
```

Comment:
Please consider a different wording:
```suggestion
echo
echo *** WARNING ***
echo
echo ** If the following embed-server command fails, manual intervention is needed.
echo ** In such case, remove any <extension> and <subsystem> declarations referring
echo ** to the removed smallrye modules from the standalone-ha.xml file and rerun this script.
echo ** For details, see Migration Changes section in the Upgrading guide.
echo ** We apologize for this inconvenience.
echo
```
Same comment (with alteration of the file name) applies to `migrate-standalone.cli` below as well.